test/integration2/migrate/migration-integration.spec.js

```diff
@@ -81,6 +81,20 @@ describe('Migrations', function () {
             });
         });
 
+        it.only('should not fail alter-table-ignore-case-sensitive', async () => {
+          return knex.migrate
```

Comment:
* please rewrite it to use async await
* please add assertion, which would only pass if both migrations completed successfully

Reply:
Thanks, updated test case as requested